From 58c9b1fb0da04f39fa66eef04534a52f07a1883d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 25 Jan 2024 12:36:32 +0100
Subject: [PATCH] theme: fix NaApplet class name

This is a property of NaApplet, not NaTrayApplet. Also increase the padding a bit to give more space between the status icons.
---
 data/theme/common.css | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/theme/common.css b/data/theme/common.css
index 109678e52..d337e0526 100644
--- a/data/theme/common.css
+++ b/data/theme/common.css
@@ -58,7 +58,7 @@ panel-toplevel.vertical gp-menu-button .icon {
 }
 
 na-tray-applet {
-  -NaTrayApplet-icon-padding: 2;
+  -NaApplet-icon-padding: 4;
 }
 
 gp-applet.separator > separator {
-- 
2.43.0

